README.md

```diff
@@ -227,90 +253,90 @@ $ sudo python3 -m pip install git+https://github.com/WebThingsIO/gateway-addon-p
 
 ## Download and Build Gateway
 
-* Clone the GitHub repository (or fork it first):
+- Clone the GitHub repository (or fork it first):
 
-    ```
-    $ git clone https://github.com/WebThingsIO/gateway.git
-    ```
+  ```
+  $ git clone https://github.com/WebThingsIO/gateway.git
+  ```
 
-* Change into the gateway directory:
+- Change into the gateway directory:
 
-    ```
-    $ cd gateway
-    ```
+  ```
+  $ cd gateway
+  ```
 
-* If you have chosen to install nvm above, install and use an LTS version of node and then set the default version. The **`.nvmrc`** file will be used by nvm to determine which version of node to install.
+- If you have chosen to install nvm above, install and use an LTS version of node and then set the default version. The **`.nvmrc`** file will be used by nvm to determine which version of node to install.
 
-    ```
-    $ nvm install
-    $ nvm use
-    $ nvm alias default $(node -v)
-    ```
+  ```
+  $ nvm install
+  $ nvm use
+  $ nvm alias default $(node -v)
+  ```
 
-* Verify that node and npm have been installed:
+- Verify that node and npm have been installed:
 
-    ```
-    $ node --version
-    v10.19.0
-    $ npm --version
-    6.14.0
-    ```
+  ```
+  $ node --version
+  v12.19.0
+  $ npm --version
+  6.14.8
+  ```
 
-    Note: these versions might differ from the LTS version installed locally.
+  Note: these versions might differ from the LTS version installed locally.
 
-* Install dependencies:
+- Install dependencies:
 
-    ```
-    $ npm ci
-    ```
+  ```
+  $ npm ci
+  ```
 
-* Add Firewall exceptions (Fedora Linux Only)
+- Add Firewall exceptions (Fedora Linux Only)
 
-    ```
-    $ sudo firewall-cmd --zone=public --add-port=4443/tcp --permanent
-    $ sudo firewall-cmd --zone=public --add-port=8080/tcp --permanent
-    $ sudo firewall-cmd --zone=public --add-port=5353/udp --permanent
-    ```
+  ```
+  $ sudo firewall-cmd --zone=public --add-port=4443/tcp --permanent
+  $ sudo firewall-cmd --zone=public --add-port=8080/tcp --permanent
+  $ sudo firewall-cmd --zone=public --add-port=5353/udp --permanent
+  ```
 
+- Set up domain:
+  _ If you plan to use Mozilla's provided tunneling service to set up a `_.mozilla-iot.org` domain: \* Start the web server:
```

Comment:
This ended up messing up the formatting of the whole section. Guessing you meant to start the line with `-` rather than `_`.